_toc.yml

```diff
@@ -0,0 +1,61 @@
+format: jb-book
+root: README
+parts:
+  - caption: Getting started
+    chapters:
+    - file: notebooks/START_HERE
+  - caption: Failure type classifier
+    chapters:
+    - file: notebooks/failure-type-classification/testgrid_flakiness_detection
+    - file: notebooks/failure-type-classification/failure_type_classifier
+    - file: notebooks/failure-type-classification/failure_type_functions
+    - file: notebooks/failure-type-classification/model_seldon
+  - caption: Time to merge prediction
```

Comment:
Same as above comment, the ml project readme could be included here

Reply:
++